sys/fido2/ctap/transport/hid/Kconfig

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2021 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menuconfig MODULE_FIDO2_CTAP_TRANSPORT_HID
+    bool "FIDO2 CTAP TRANSPORT HID"
+    depends on TEST_KCONFIG
```

Comment:
Should depend on `USEMODULE_FIDO2_CTAP_HID` or `USEMODULE_FIDO2_CTAP_TRANSPORT_HID` after adding "transport" to the module name